esp-wifi/src/compat/common.rs

```diff
@@ -129,18 +122,6 @@ pub fn sem_create(max: u32, init: u32) -> *mut c_void {
 pub fn sem_delete(semphr: *mut c_void) {
     trace!(">>> sem delete");
 
-    // TODO remove this once fixed in esp_supplicant AND we updated to the fixed -
-    // JIRA: WIFI-6676
-    unsafe {
```

Comment:
Where did we update to the fixed supplicant?

Reply:
we didn't - but sem_delete and mutex_delete just call free so it's "fine"